utilities/ldap_sync.sql

```diff
@@ -237,35 +1023,45 @@ get_ad_group_members AS (
 					SELECT distinct  DISTINGUISHED_NAME 
 					FROM
 		 			EXA_DBA_USERS
-				)
+				) and VAL NOT like '%No such object%'
 			)  --get uid attribute as USER_NAME in database
 		
-		)WHERE VAL NOT like '%No such object%'
-		AND UPPER(VAL) NOT IN (SELECT VAL FROM ALTER_USERS)
+		)
+		where UPPER(VAL) NOT IN (SELECT VAL FROM ALTER_USERS)
 
 	)
+---------------------------------------
 ,revokes AS (
--- This CTE will only revoke roles from users if they are a part a member of the role in EXA, but are no longer in the group in LDAP
+---------------------------------------
+/* This CTE will only revoke roles from users if they are a part a member of the role in EXA, but are no longer in the group in LDAP */
+
 		SELECT 'REVOKE "' || GRANTED_ROLE || '" FROM "' || UPPER(GRANTEE) || '";' AS DCL_STATEMENT, 3 ORDER_ID from exa_membership e
                 full outer join get_ad_group_members a on e.role_comment = a.search_string and e.distinguished_name = a.val
```

Comment:
The logic in the comment above together with the existence of the filter in this subquery suggests that it isn't FULL but LEFT or RIGHT join.